packages/mongo-nav/src/user-menu/UserMenu.spec.tsx

```diff
@@ -54,26 +54,20 @@ describe('packages/mongo-nav/user-menu', () => {
   });
 
   test('renders university MenuItems when university dropdown is clicked and closes other SubMenus', () => {
-    const { getByTestId } = renderUserMenu({
+    const { getByTestId, getAllByRole, findByTestId } = renderUserMenu({
       activePlatform: 'cloud',
     });
     const trigger = getByTestId('user-menu-trigger');
     fireEvent.click(trigger);
 
-    const university = document.querySelectorAll(
-      '[data-leafygreen-ui="sub-menu-container"]',
-    )[1];
-
-    const universityArrowButton = university?.parentNode?.querySelector(
-      'button',
-    );
+    const universityTrigger = getAllByRole('button', {
+      name: 'Open Sub-menu',
+    })[1];
```

Comment:
I think this test is failing. I could see it being more stable if the button was selected based on its text content.

Reply:
Great call, fixed! Thank you!